keps/sig-node/2400-node-swap/README.md

```diff
@@ -354,6 +390,18 @@ and/or workloads in a number of different scenarios.
 Since swap provisioning is out of scope of this proposal, this enhancement
 poses low risk to Kubernetes clusters that will not enable swap.
 
+#### Existing use cases of Swap
+
+As beta2 was being worked on, we discovered use cases where `--fail-swap-on=false` is used but Kubernetes is not utilizing swap.
+Kind e2e tests run kubelet with `--fail-swap-on=false` and
+the default developer configuration for `hack/local-up-cluster` allows for running developer clusters with swap enabled.
+
+We need to support the `--fail-swap-on=false` for both cgroup v1 and cgroupv2. We will not support KEP-2400 with cgroup v1.
+So when one wants to GA this feature, we need to have a way to disable workloads from using swap while keeping the feature toggle on.
+To address this, we will propose a new field to `MemorySwap` called `NoSwap`. This will disable swap usage on the node while keeping the feature active.
```

Comment:
Setting `NoSwap` will disable swap for workloads.
OTOH this could mean that if swap is present on a node, but NoSwap is used, then swap is still being used for the system daemons (assuming the correct systemd configuration).

This is just an observation as there was also a discussion about how to enable swap for system daemons (I don't recommend this).

Reply:
That is a good point. I should add that to the PRR around how to roll back this change.

Reply:
I realize that was actually added in the rollback section.